Source/WebKit/WPEPlatform/wpe/WPEView.h

```diff
@@ -96,11 +96,13 @@ typedef enum {
 /**
  * WPEViewError:
  * @WPE_VIEW_ERROR_RENDER_FAILED: Failed to render
+ * @WPE_VIEW_ERROR_INITIALIZATION_FAILED: Failed to initialize rendering
```

Comment:
This is weird, because in the base api there's no api to initialize rendering, I guess we can just use render error in the end.

Reply:
You're right -- that category would only be used by the internal QtQuick view/display - makes no sense to introduce that type - will remove it.